.babelrc

```diff
@@ -5,6 +5,7 @@
   ],
   "plugins": [
     "transform-async-to-generator",
-    "babel-plugin-transform-runtime"
+    "babel-plugin-transform-runtime",
+    "babel-plugin-transform-object-rest-spread"
```

Comment:
We can update the preset to es2018 and remove this additional plugins, but it can be done afterwards.